ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/card/card.tsx

```diff
@@ -45,8 +46,9 @@ const Details = styled.div`
 const Footer = styled.div`
   height: ${props => props.theme.minimumButtonSize};
   text-align: right;
-  display: flex;
-  justify-content: flex-end;
+  position: absolute;
```

Comment:
does this work fine with different zoom levels and spacing?

Reply:
We might be better off doing this via flexbox and `space-between` on the root element instead.

Reply:
Refactored to flexbox!